workshops/filters.py

```diff
@@ -207,6 +211,139 @@ def get_order_by(self, order_value):
         return super().get_order_by(order_value)
 
 
+def filter_all_persons(queryset, all_persons):
+    if all_persons:
```

Comment:
Please add a docstring -- I understood the purpose of this function after reading `all_persons` label.